src/styles.scss

```diff
@@ -4,12 +4,15 @@ $fa-font-path : '../node_modules/font-awesome/fonts';
 html,
 body {
     font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
-    max-width: 1540px;
-    margin: 0 auto;
-    height: 100%;
+    margin: 0;    
```

Comment:
I know a handful of apps internally have max widths and gutters on the side. I'm not sure if we need to replicate that functionality in cashmere's docs or not.